config.rb

```diff
@@ -108,7 +108,7 @@ def self.test?
 
   # Postgres
   optional :postgres_service_project_id, string
-  optional :postgres_service_hostname, string
+  override :postgres_service_hostname, "postgres.ubicloud.com", string
   optional :postgres_service_blob_storage_access_key, string
   optional :postgres_service_blob_storage_secret_key, string
```

Comment:
Should we set `clear: true` for them?

Reply:
Added for secret key. Access key does not need to be cleared.